lib/liquid/combinators/tags/for.ex

```diff
@@ -51,6 +51,21 @@ defmodule Liquid.Combinators.Tags.For do
   alias Liquid.Combinators.{General, Tag}
   alias Liquid.Combinators.Tags.Generic
 
+  @type t :: [
+          for: [
+            for_statements: [
+              variable: String.t(),
+              value: LexicalToken.value(),
+              for_params: [
+                offset: Integer.t() | String.t(),
```

Comment:
offset and limit are optional, must be `offset: ...| limit:...` instead of `ofset:..,limit:...`